drivers/timer/mcux_lptmr_timer.c

```diff
@@ -81,6 +81,11 @@ uint32_t sys_clock_cycle_get_32(void)
 	return LPTMR_GetCurrentTimerCount(LPTMR_BASE) + cycles;
 }
 
+uint64_t sys_clock_cycle_get_64(void)
+{
+	return 0;
```

Comment:
@henrikbrixandersen   - would be great to get your input here. It seems there is no 64-bit timer register on this platform.

For software emulation, I would like to increment a 32-bit "top half" each time a 32-bit cycle counter overflows (assuming a 1:1 ratio), shift and then OR it with the regular 32-bit value.

In this driver, where is the best location to increment the 32-bit "top half"?